### PR TITLE
Add test classes to improve Coverage

### PR DESCRIPTION
**Pull request by kevinswk94**
Do not merge, still a work in progress

### CODE REVIEW

**Comment by muhd97** [comment]
Noted

File: src/test/java/seedu/address/testutil/RecipeBuilder.java [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+ * A utility class to help with building Recipe objects.
+ */
+public class RecipeBuilder {
+
+    public static final String DEFAULT_NAME = "Ham Sandwich";
+    public static final String DEFAULT_INGREDIENT_LIST = "bread, 2 slices; ham, 1 slice";
+    public static final String DEFAULT_INSTRUCTION_LIST = "place ham between slices of bread; eat";
+
+    private Name name;
+    private IngredientList ingredients;
```
Comment (muhd97) [inline src/test/java/seedu/address/testutil/RecipeBuilder.java]: Need to have one more attribute.
private Calorie calorie

File: src/test/java/seedu/address/logic/commands/CommandTestUtil.java [hunk 2]
```diff
@@ -2,9 +2,9 @@
 
 import static org.junit.jupiter.api.Assertions.assertEquals;
 import static org.junit.jupiter.api.Assertions.assertTrue;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_INGREDIENTS;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_INSTRUCTIONS;
```
Comment (muhd97) [inline src/test/java/seedu/address/logic/commands/CommandTestUtil.java]: There is one more prefix now that u need to include for calorie

**Review by muhd97 — DISMISSED** [review]
👍 great job!!

**Review by AdarshChugani — APPROVED** [review]
Looks Good!

Thanks Kev!

**Pull request merged**